getting-started/macos/macos-basic-setup.md

```diff
@@ -0,0 +1,80 @@
+---
+ID:
+post_title: Basic Roots development environment for macOS
+author: Michael W. Delaney
+post_excerpt: ""
+layout: doc
+permalink: >
+
+published: false
+post_date:
+---
+
+Basic macOS 10.13+ development environment for Roots Trellis, Bedrock, and Sage
+
+## Requirements
+
+Roots Trellis, Bedrock, and Sage development on macOS is supported by several libraries and software packages. This guide will show you how to install and configure each of these recommended packages on your computer.
+
+- Xcode Command Line Tools
+- Git
+- Homebrew
+- Composer
+- nvm
+- Node.js
+
+## Xcode Command Line Tools
+
+Install Xcode Command Line Tools using the built-in command:
+
+    $ xcode-select --install
+
+## Git
+
+Git is automatically installed as part of the Xcode Command Line tools. Configure Git with your user information:
+
+    $ git config --global user.name "Your Name"
+    $ git config --global user.name "yourname@example.com"
+
+## Homebrew
+
+Install Homebrew from the instructions at [https://brew.sh](https://brew.sh/), or with the following command:
+
+    $ /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+
+Install Cask:
+
+    $ brew tap caskroom/cask
+
+## Composer
+
+Install Composer from Homebrew:
+
+    $ brew install composer
+
+# SSH Keys
+
+---
+
+## Create SSH Keys
+
+Trellis and GitHub both use SSH keys to communicate securely without the need to type a username and password each time. Create your SSH keys and add them to your macOS Keychain:
```

Comment:
Adding them is covered under the next section, so I'd suggest removing the language here to avoid confusion.

Reply:
This language is here to present what they're about to read. Sort of a "here's what you're about to do". It's a little longer than the other introductions, like "install homebrew" but that's the intent. Does that help,  or does it still seem redundant?